server/rooms.js

```diff
@@ -591,37 +591,55 @@ class GlobalRoom extends BasicRoom {
 		}
 		return this.formatList;
 	}
-	get configRankList() {
-		if (Config.nocustomgrouplist) return '';
-
-		// putting the resultant object in Config would enable this to be run again should config.js be reloaded.
-		if (Config.rankList) {
-			return Config.rankList;
-		}
-		let rankList = [];
-
-		for (let rank in Config.groups) {
-			if (!Config.groups[rank] || !rank) continue;
-
-			let tarGroup = Config.groups[rank];
-			let groupType = tarGroup.addhtml || (!tarGroup.mute && !tarGroup.root) ? 'normal' : (tarGroup.root || tarGroup.declare) ? 'leadership' : 'staff';
-
-			rankList.push({symbol: rank, name: (Config.groups[rank].name || null), type: groupType}); // send the first character in the rank, incase they put a string several characters long
+	/**
+	 * @param {AnyObject} group
+	 */
+	configGroupListMapper(group) {
```

Comment:
This seems to be hiding a `getGroupType` method inside of it

Reply:
Already discussed this in the dev room, the problem was more a matter of naming